compiler-plugin/src/main/scala/scalapb/compiler/ProtoValidation.scala

```diff
@@ -103,10 +103,17 @@ class ProtoValidation(implicits: DescriptorImplicits) {
           s"${m.getFullName}: sealed oneofs may not contain nested enums"
         )
       }
-    } else if (m.sealedOneOfExtendsCount > 0) {
-      throw new GeneratorException(
-        s"${m.getFullName}: is not a Sealed oneof and may not contain a sealed_oneof_extends message option. Use extends instead."
-      )
+    } else {
+      if (m.sealedOneOfExtendsCount > 0) {
+        throw new GeneratorException(
+          s"${m.getFullName}: is not a Sealed oneof and may not contain a sealed_oneof_extends message option. Use extends instead."
```

Comment:
s"${m.getFullName}: sealed_one_of_extend can only be used on sealed one ofs."   - (I think that the advice to use `extends` is unlikely to be relevant, it is more likely that the user misplaced the option, or assumes that something is a sealed signoff and it isn't.

Reply:
Hm. It's double-sided, I suppose. They may just missclick on the IDE suggest, for example. :)

Maybe we may elaborate better help message in that case, pointing out that maybe user wanted to use a `oneof sealed_value` and a link to the documentation section over its use.

Reply:
> Hm. It's double-sided, I suppose. They may just missclick on the IDE suggest, for example.

I think that since it's ambiguous why the user did it a error message that elaborates on the two options may be too long and confusing. I think that a one line `"sealed_one_of_extend can only be used on sealed one ofs"` would set the user in the right direction without making assumptions.